binder/environment.yml

```diff
@@ -2,34 +2,34 @@ channels:
 - conda-forge
 dependencies:
 # Jupyter
-- jupyterlab=2
-- nodejs=11
-- nbconvert=5.5
-- widgetsnbextension=3.5
+- jupyterlab=3
+- nodejs=14
+- nbconvert=6
 # Python Kernel
 - ipykernel=5.1
-- ipywidgets=7.5
-- ipyleaflet=0.12
-- altair=3.1
-- bqplot=0.12
-- dask=2.1
-- matplotlib=3.1
-- pandas=0.24
+- xeus-python=0.9
+- ipywidgets=7.6
+# - ipyleaflet=0.12
+# - altair=3.1
+# - bqplot=0.12
+# - dask=2.1
+# - matplotlib=3.1
+- pandas=1
 - python=3.7
 - scikit-image=0.15
 - scikit-learn=0.21
-- seaborn=0.9
+- seaborn=0.11
 - tensorflow=1.13
 - sympy=1.4
 - pyyaml
 - traittypes==0.2.1
 - invoke=1.2
 # C++ Kernel
-- xeus-cling=0.6
+# - xeus-cling=0.6
```

Comment:
I think we can require `xeus-cling=0.10`, `xwidgets=0.24.2`, `xleaflet=0.13.1`  which should not cause conflicts.